ape_solidity/compiler.py

```diff
@@ -378,31 +383,30 @@ def get_version_map(
         if not isinstance(contract_filepaths, (list, tuple)):
             contract_filepaths = [contract_filepaths]
 
-        contract_filepaths_set = set(contract_filepaths)
-        contracts_path = base_path or self.config_manager.contracts_folder
-        imports = self.get_imports(get_all_files_in_directory(contracts_path), contracts_path)
+        base_path = base_path or self.project_manager.contracts_folder
+        contract_filepaths_set = verify_contract_filepaths(contract_filepaths)
+        sources = [p for p in self.project_manager.source_paths if p.suffix == ".sol"]
+        imports = self.get_imports(sources, base_path)
 
         # Add imported source files to list of contracts to compile.
-        source_paths_to_compile = contract_filepaths_set.copy()
+        source_paths_to_get = contract_filepaths_set.copy()
         for source_path in contract_filepaths_set:
-            imported_source_paths = self._get_imported_source_paths(
-                source_path, contracts_path, imports
-            )
+            imported_source_paths = self._get_imported_source_paths(source_path, base_path, imports)
```

Comment:
Noticed here and was wondering about renaming the _get_imported_source_paths arg for the `base_path` instead of `contracts_path`?
<img width="603" alt="Screen Shot 2022-08-30 at 12 13 43 PM" src="https://user-images.githubusercontent.com/32708219/187487871-bb551d6e-590b-4020-b602-741ca7e0794c.png">

Reply:
Done in 0536e2b